bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/i18n/LocaleProvider.java

```diff
@@ -25,4 +28,10 @@
      */
     Locale getLocale();
 
+    /**
+     * Provides access to the location of the installation
+     *
+     * @return location of the current installation or null if the location is not set
+     */
+    PointType getLocation();
```

Comment:
`ConfigurationProvider` would definitely be too broad (that actually already exists as the OSGi Configuration Admin) and having a new LocationProvider for a single value might not be worth the effort, especially as Locale and Location seem to have something in common, if you look at the word, right? ;-)